topics/dev/tutorials/core-contributing/tutorial.md

```diff
@@ -3,15 +3,17 @@ layout: tutorial_hands_on
 
 title: "Contributing a New Feature to Galaxy Core"
 questions:
- - How can I develop extensions to Galaxy data model?
- - How can I implement new API functionality within Galaxy?
- - How can I extend the Galaxy user interface with VueJS components?
+ - How can I add a new feature to Galaxy that involves modifications to the database, the API, and the UI?
 objectives:
+ - Learn to develop extensions to the Galaxy data model
+ - Learn to implement new API functionality within Galaxy
+ - Learn to extend the Galaxy user interface with VueJS components
 time_estimation: "3H"
 contributors:
  - jmchilton
+ - jdavcs
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Reply:
:point_up: @jmchilton up to you. I'm fine with either way.